package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Utilities to have an async fetch function in Remix apps",
   "engines": {
     "node": ">= 18",
-    "pnpm": ">= 9"
+    "pnpm": ">= 8"
```

Comment:
For a library, we don't need to constrain the package manager version. For apps, I like to do it so that people get warnings if they're not on the same version as everyone else.